File: practice-code-blue/programme-c/fibonacci.c
#include<stdio.h>

int fibonacci(int n)
{
 if(n==0)
 {
 return 0;
 }
 if(n==1)
 {
  return 1;
 }
 return fibonacci(n-1) + fibonacci(n-2);
}

int main()
{
int i;
for(i=1;i<=10;i++)
{
  printf("%d ",fibonacci(i));
}

printf("\n");
}

File: java-programs/loops/fibonacci.java
import java.io.*;
import java.util.Scanner;

class fibonacci
{      
    public static void main(String args[])
    {
     int i,n,a = 0,b = 1,c = 0;
     n = 6;

     System.out.print(" "+a+" "+b);  
     
     for(i=1;i<=(n-2);i++)
     {
      c = a + b;
      System.out.print(" "+c);
      a = b;
      b = c;
     }
      System.out.println();
    }
}